wsclient/pom.xml

```diff
@@ -131,7 +131,7 @@
     <dependency>
       <groupId>org.hpccsystems</groupId>
       <artifactId>commons-hpcc</artifactId>
-      <version>7.6.10-SNAPSHOT</version>
```

Comment:
You shouldn't change the version number, you might need to rebase your branch against master

Reply:
@rpastrana updated